test/createAzureResource.test.ts

```diff
@@ -93,6 +94,26 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         assert.equal(createdApp.state, 'Running', `Web App state should be 'Running' rather than ${createdApp.state}.`);
     });
 
+    test('Configure Deployment Source to LocalGit', async () => {
+        let createdApp: WebSiteManagementModels.SiteConfigResource;
```

Comment:
You can just do `let createdApp: WebSiteManagementModels.SiteConfigResource = await webSiteClient.webApps.getConfiguration(resourceName, resourceName);` rather than separating it out in two lines.